l1-contracts/contracts/bridgehub/InteropCenter.sol

```diff
@@ -0,0 +1,400 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+// solhint-disable reason-string, gas-custom-errors
```

Comment:
this is a new contract, so please remove this line to not forget about custom errors